examples/vello_editor/src/text.rs

```diff
@@ -350,6 +466,13 @@ impl Editor {
         if let Some(cursor) = self.selection.focus().weak_geometry(&self.layout, 1.5) {
             scene.fill(Fill::NonZero, transform, Color::LIGHT_GRAY, None, &cursor);
         };
+        if let ComposeState::Preedit { text_at } = self.compose_state {
```

Comment:
Given that this will always do a relayout if this is changing, can we just add a `Style::Underline` to this region when creating the layout? That's the solution we use in Masonry.

Reply:
Thank you, yes, that works great. I hadn't realized underlining was part of Parley, but that makes sense!